app/models/position.rb

```diff
@@ -16,7 +16,13 @@ class Position < ActiveRecord::Base
   #  obj.save
   #end
 
-  scope :from_role, -> role { joins(:role).where(acl_role: { acl_role_name: role })}
+  scope :from_role, -> role { joins(:role).where(acl_role: { acl_role_name: role }) }
+
+  scope :for_phonebook, -> {
+    joins(:role, :user)
+    .where('acl_role.acl_role_id NOT IN (34,21,37,5,31,35,36)')
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.